cmd/serve.go

```diff
@@ -93,5 +99,6 @@ func init() {
 	serveCmd.PersistentFlags().StringArrayVar(&ignoredUserAgents, "ignored-user-agent", []string{}, "user agents to ignore (reply with empty body and 200 OK). A user agent is ignored if it contains the one of the values passed to this flag")
 	serveCmd.PersistentFlags().DurationVar(&cacheDuration, "cache-duration", time.Minute*1, "how long to keep a generated page in cache before expiring it, 0 to never expire")
 	serveCmd.PersistentFlags().DurationVar(&refreshInterval, "refresh-interval", time.Minute*15, "how long to wait before trying to get fresh data from the target registry")
+	serveCmd.PersistentFlags().IntVar(&requestsPerSecond, "requests-per-second", 1, "limit the number of requests per second that can be done to the target registry")
```

Comment:
Is one per second a good default? It seems a very low rate.

Reply:
yes it's low but I tested it locally against wave community and it feels fast enough. The async registry is updating the internal registry representatation in background so it's an issue visible only on restart.

Reply:
Agreed that rate limiter is one I also like, felt like it was easy enough to avoid a dependency however.